cloudflare.install

```diff
@@ -38,3 +36,13 @@ function cloudflare_requirements($phase) {
   }
   return $requirements;
 }
+
+/**
+ * Installs ctools module.  Uninstalls cloudflare_zone_ui.
+ */
+function cloudflare_update_8001(&$sandbox) {
```

Comment:
add a drupal set message to tell users to remove cloudflare_zone_ui because it's deprecated and will be removed in beta-2